plugins/datadog-entity-sync-node/src/transforms/defaultEntitySerializer.ts

```diff
@@ -0,0 +1,199 @@
+import gitUrlParse from 'git-url-parse';
+
+import type {
+  ComponentEntity,
+  ApiEntity,
+  SystemEntity,
+  ResourceEntity,
+  Entity,
+} from '@backstage/catalog-model';
+import {
+  ANNOTATION_SOURCE_LOCATION,
+  getCompoundEntityRef,
+  getEntitySourceLocation,
+  isComponentEntity,
+  isApiEntity,
+  isSystemEntity,
+  isResourceEntity,
+  parseEntityRef,
+  RELATION_OWNED_BY,
+  stringifyEntityRef,
+} from '@backstage/catalog-model';
+import { TECHDOCS_ANNOTATION } from '@backstage/plugin-techdocs-common';
+
+import { valueGuard } from '@cvent/backstage-plugin-datadog-entity-sync-node';
+
+function ensureComponent(
```

Comment:
Should probably change the name, as it's no longer `components`, maybe `ensureEntity`? 🤷